test/ex_doc/formatter/html_test.exs

```diff
@@ -85,6 +85,8 @@ defmodule ExDoc.Formatter.HTMLTest do
   end
 
   test "multiple extras with the same name", c do
+    import LazyHTML
```

Comment:
We do just a few function calls, so I would say an import is not necessary. In general it's easier to understand code with qualified calls, unless it's well-established macros where `import` makes sense :)